src/ansiblelint/__main__.py

```diff
@@ -131,13 +131,20 @@ def report_outcome(result: "LintResult", options, mark_as_success=False) -> int:
             break
     msg += "".join(sorted(entries))
 
-    for k, v in used_old_tags.items():
-        _logger.warning(
-            "Replaced deprecated tag '%s' with '%s' but it will become an "
-            "error in the future.",
-            k,
-            v,
-        )
+    # Do not deprecate the odl tags just yet. Why? Because it is not currently feasible
+    # to migrate old tags to new tags. There are a lot of things out there that still
+    # use ansible-lint 4 (for example, Ansible Galaxy and Automation Hub imports). If we
+    # replace the odl tags, those tools will report warnings. If we do not replace them,
```

Comment:
@felixfontein You missed this:)
```suggestion
    # replace the old tags, those tools will report warnings. If we do not replace them,
```